README.md

```diff
@@ -55,7 +55,7 @@ To hand in and get feedback on your iterations you must follow this process for
 - In the github interface find your personal branch and make a pull request towards the `iteration-1` branch. Share the link with your team member(s) and ask for feedback.
```

Comment:
The naming of the iteration branch will change over time. E.g we will have iteration-1, iteration-2 and iteration-3 . The Description should probably reflect this.